brave/brave-core
# Allows to make IAP for Android Leo

Resolves https://github.com/brave/brave-browser/issues/34264
Resolves https://github.com/brave/brave-browser/issues/35924



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Go to Settings->Leo
2. Go to Go Premium
3. Observe the purchase is done.

![Screenshot_20240208_112625](https://github.com/brave/brave-core/assets/12011303/5d7fb0a9-2266-41c7-acc4-a18ae5600a72)
